src/content/advanced/gitops/environments/index.md

```diff
@@ -0,0 +1,401 @@
+  minSize: 5
+  name: machine-prod-pool0
+  rootVolumeSizeGB: 300
+network:
+  availabilityZoneUsageLimit: 3
+```
+
+We want now to set an different version for some default apps installed in the cluster (for example, we want to use a known stable version). Like in dev environment, we create a file `default_apps_config.yaml` with the specific values.
+
+```yaml
+userConfig:
+  apps:
+    cilium:
+      version: 0.6.0
+    coreDNS:
+      version: 1.12.0
+```
+
+It is similar to the development cluster in the following manners:
+
+- We are using a base and modifying values with an overlay configuration.
+- We can change values in the cluster config and in the default apps.
+
+The final structure of our stages is:
+
+```nohighlight
+bases/environments/stages/
+├── dev
+│   └── example_cluster
+│       ├── cluster_config.yaml
+│       ├── default_apps_config.yaml
+│       └── kustomization.yaml
+└── prod
+    └── example_cluster
+        ├── cluster_config.yaml
+        ├── default_apps_config.yaml
+        └── kustomization.yaml
+```
+
+> In a similar way, we can base our stages in different cluster templates that we create from bases. We can create as many levels as we want, always keeping in mind properly setting the priorities of the config files.
+
+#### Region specific settings for the production cluster
+
+Let's create some bases for our region setup. Relative to root of the repository let's execute the following commands.
+
+```nohighlight
+mkdir -p bases/environments/regions/eu_central
+mkdir -p bases/environments/regions/us_west
+
+cd bases/environments/regions
+```
+
+For the `eu-central` region, we are creating a `cluster_config.yaml` file and a `kustomization.yaml` file. 
+
+`cluster_config.yaml`
+
+```yaml
+controlPlane:
+  availabilityZones:
+    - eu-central-1
+    - eu-central-2
+    - eu-central-3
+nodeCIDR: "10.32.0.0/24"
+```
+
+> These values are examples and need to be replaced by real values of the user account.
+
+`kustomization.yaml`
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+buildMetadata: [originAnnotations]
+configMapGenerator:
+  - files:
+    - values=cluster_config.yaml
+    name: ${cluster_name}-region-config
+    namespace: org-${organization}
+generatorOptions:
+  disableNameSuffixHash: true
+patches:
+  - patch: |
+      - op: add
+        path: /spec/extraConfigs/0
+        value:
+          - name: "${cluster_name}-region-config"
+            namespace: org-${organization}
+            priority: 120
+    target:
+      group: application.giantswarm.io
+      kind: App
+      name: ${cluster_name}
+      namespace: org-${organization}
+kind: Kustomization
+```
+
+With these files we are creating a new `configmap` and adding it as an extra config in the cluster app that will create the cluster resources.
+
+Now, for the `us-west` region we create a `cluster_config.yaml` file with different configuration. We can copy the `kustomization.yaml` file from the other region as it will be the same.
```

Comment:
```suggestion
With these files, we are creating a new `configmap` and adding it as an extra config in the cluster app that will create the cluster resources.

Now, for the `us-west` region we create a `cluster_config.yaml` file with a different configuration. We can copy the `kustomization.yaml` file from the other region as it will be the same.
```